packages/core/README.md

```diff
@@ -0,0 +1,69 @@
+# Core
+
+[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
+[![Powered by Mason](https://img.shields.io/endpoint?url=https%3A%2F%2Ftinyurl.com%2Fmason-badge)](https://github.com/felangel/mason)
+[![License: MIT][license_badge]][license_link]
+
+Core components for Widgetbook Demo App
+
+## Installation 💻
```

Comment:
This is generated. Shall we adjust this now or later? If we do it later how do we not forget about it?

Reply:
I added some initial description to the README bb57ba9ee0105cef70293a3ec7fca4a8a5690b95. Please let me know what you think. I will need to make some folder structure changes later to match the structure in the README (and the task) and would be happy to discuss this further.